docs/components/movement-sensor/wheeled-odometry.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Configure a wheeled-odometry movement sensor"
+linkTitle: "wheeled-odometry"
+type: "docs"
+description: "Configure a wheeled-odometry movement sensor."
+images: ["/icons/components/imu.svg"]
+tags: ["movement sensor", "components", "encoder", "motor", "base", "wheeled", "odometry"]
+# SMEs: Rand, Martha
+---
+
+Configure a `wheeled-odometry` movement sensor to implement _wheeled odometry_ on your robot.
+
+_Wheeled odometry_ is the estimation of the rate of change of position, orientation, linear velocity, and angular velocity using the dimensions of a base, calculated by measuring the movement of the motors through encoders.
+Because of this method of estimation, you don't have to have a specific piece of movement sensor hardware to implement `wheeledodometry` on your robot. 
```

Comment:
```suggestion
Because of this method of estimation, you don't have to have a specific piece of movement sensor hardware to implement `wheeled-odometry` on your robot. 
```